src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1470,7 +1470,10 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Predefined type '{0}' is not defined or imported</value>
   </data>
   <data name="ERR_PredefinedValueTupleTypeNotFound" xml:space="preserve">
-    <value>Predefined type '{0}' is not defined or imported, or is declared in multiple referenced assemblies</value>
+    <value>Predefined type '{0}' is not defined or imported</value>
+  </data>
+  <data name="ERR_PredefinedValueTupleTypeAmbiguous3" xml:space="preserve">
+    <value>Predefined type '{0}' is declared in multiple referenced assemblies including '{1}' and '{2}'</value>
```

Comment:
Minor point: `including` may imply there are more than two assemblies which is unlikely. Perhaps shorten to `declared in multiple assemblies: '{1}' and '{2}'`. See `ERR_TypeForwardedToMultipleAssemblies`.